crates/pessimistic-proof-test-suite/src/bin/ppgen.rs

```diff
@@ -89,13 +89,14 @@ pub fn main() {
         .generate_plonk_proof(&old_state.into(), &multi_batch_header)
         .expect("proving failed");
     let duration = start.elapsed();
-
     info!(
         "Successfully generated the plonk proof with a latency of {:?}",
         duration
     );
 
     let vkey = vk.bytes32().to_string();
+    info!("vkey: {}", vkey);
```

Comment:
left over debug?

Reply:
Okey, at the end it would be better to have a dedicated CLI command on `agglayer` to retrieve it directly